code/modules/WW2/_mags.dm

```diff
@@ -24,6 +25,7 @@ WW 2 WEAPONS MAGS N AMMO
 
 /obj/item/ammo_magazine/mosinbox
 	name = "magazine box (7.62x54mm)"
+	desc = "this is a box with 7,62c54mm that is used for Mosin clips to get refilld"
 	icon_state = "mosin_ammo"
 //	origin_tech = "combat=2"
```

Comment:
You guys don't use `origin_tech` anymore this should honestly just be deleted.

Reply:
ter is no reason to remove it any way just gives us extra work

Reply:
>deleting a few commented out lines is extra work

👌

Reply:
good that you understand

Reply:
absolute state of leben dev team